packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -258,8 +258,8 @@
     &-container {
         position: relative;
         margin-top: 0.8rem;
-        min-width: 15rem;
-        width: 100%;
+        min-width: 43rem;
+        width: fit-content;
```

Comment:
These changes are affecting all the dropdown boxes in Deriv app, you can definitely see different width for dropdowns in **Account > Finanacial assessment** compared to Production.
One better way might be to use `text-overflow: ellipsis`.
I suggest looking into this article to find a better solution https://css-tricks.com/handling-long-unexpected-content-css/